src/create.wsgi

```diff
@@ -183,22 +184,21 @@ def application(environ, start_response):
         return response(start_response, "500 Internal Server Error",
                         _("Unable to unpack archive"))
 
-    os.unlink(archive.name)
+    Path(archive.name).unlink()
 
-    files = os.listdir(crashdir)
+    files = list(crashdir.iterdir())
```

Comment:
What does `iterdir()` return? An iterator? It would be wasteful to go this way in that case.